### PR TITLE
Saying Hello to Everyone

### PR DESCRIPTION
**Pull request by Naman-Bhalla**
This change says Heyaa as requested by the students in the class. This changes resolves bug #

### CODE REVIEW

File: hello.txt [hunk 1]
```diff
@@ -16,3 +16,4 @@ jdwqbhoihwdeihjdweo
 hello
 hi
 hello ji
+heyyaaa
```
Comment (Naman-Bhalla) [inline hello.txt]: Correct spelling

File: hello.txt [hunk 2]
```diff
@@ -16,3 +16,4 @@ jdwqbhoihwdeihjdweo
 hello
 hi
 hello ji
```
Comment (Naman-Bhalla) [inline hello.txt]: Only english

**Review by yogie-7798 — APPROVED** [review]

**Review by RakeshJakka5158 — APPROVED** [review]

**Review by SumangalapatilVijay — APPROVED** [review]

**Review by razat-thapar — APPROVED** [review]
looks good

**Review by drahulkumar — APPROVED** [review]

**Review by binaryWar — APPROVED** [review]

**Review by Anurag-Deshmukh-dev — APPROVED** [review]

**Review by abhinav11411141 — APPROVED** [review]

**Review by abhi621 — APPROVED** [review]

**Review by perfectprocrastinator — APPROVED** [review]
lgtm

**Review by Abhishekmishra-1 — COMMENTED** [review]
Good

**Pull request merged**